src/puppetlabs/ezbake/core.clj

```diff
@@ -699,6 +707,38 @@ Additional uberjar dependencies:
 (defmulti action
   (fn [action & args] action))
 
+(defn munge-deps
```

Comment:
would be lovely to add a docstring here to describe what the function actually does